orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/orca/deploymentmonitor/models/DeploymentMonitorDefinition.java

```diff
@@ -23,6 +23,7 @@
   private String id;
   private String name;
   private String supportContact;
+  private boolean isStable;
```

Comment:
Do we feel like these data classes should have docs? What are the side effects of a deployment monitor being stable or not?

Reply:
that's a good point, it's documented here https://github.com/spinnaker/orca/blob/master/orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/config/DeploymentMonitorDefinition.java but i will replicate it, or at least link to the other file